client/plots/sampleScatter.js

```diff
@@ -1111,14 +1111,14 @@ class Scatter {
 							}
 							qgroups.push(qgroup)
 						}
-
 						config.divideBy = {
 							term: { name: 'groups', type: 'samplelst', values },
 							q: {
 								mode: 'custom-groupsetting',
 								groups: qgroups
 							}
 						}
+						config.settings.matrix.colw = 10
```

Comment:
can you make it dynamically decide width, rather than hardcoding? see if matrix has an attribute to do it

Reply:
setting this property to 0 does that, but when the number of samples increases, say to 120 samples, is looking like this. 
<img width="1033" alt="Screenshot 2023-03-21 at 2 13 34 PM" src="https://user-images.githubusercontent.com/12271391/226720386-be49ab6d-1593-4f0c-b34c-9c42e942ace2.png">
And if the width is below 8 it does not show the sample id. So I think with 10 should work for most cases, groups are not that big.

Reply:
Also the sampleids are not too small or too big with this size.

Reply:
@airenzp you may want to improve the `setAutoDimensions()` function in matrix.js, instead of hardcoding.